### ISSUE TITLE
asciidoctor.Asciidoctor

### ISSUE THREAD

**Issue by niahoo** [fboulay/docpad-plugin-asciidoc#2]
Hello, 
I have a problem with you plugin. It won't run because of this :

```
TypeError: asciidoctor.Asciidoctor is not a function
    at AsciidocPlugin.render (E:\minsrc\docpadtest\node_modules\docpad-plugin-asciidoc\out\asciidoc.plugin.js:34:37)
```

The following line seems to work and returns an object :
```
asciidoctor = require('asciidoctor.js')();
```

Please see this issue : https://github.com/asciidoctor/asciidoctor.js/issues/269 . There is something wrong with the versions.

Thank you !

**Comment by fboulay** [comment]
Thanks for reporting. I will check this as soon as I can.

The asciidoctor.js version used in the docpad plugin is 1.5.5-2.
The link you are referring to indicates that the error stated to occur when using version 1.5.5-4.

I did not make any tests yet to reproduce the issue, but are you overriding the asciidoctor.js version in your package.json ?

**Comment by niahoo** [comment]
I tried overriding and not, but npm will only install 5-4 ... I'm investigating why, since your package.json is OK.

**Comment by niahoo** [comment]
Ok, it's because your package.json says `"asciidoctor.js": "~1.5.5-2"`, So it allows any `1.5.X` version.
I changed to `"asciidoctor.js": "1.5.5-2"` and it's OK.

**Comment by fboulay** [comment]
Ok great, then I'll close the issue

**Comment by niahoo** [comment]
Yeah but you may want to update package.json or anyone else will have the same problem :)

**Comment by fboulay** [comment]
@niahoo I released the version 2.2.0 which should fix your problem

**Comment by niahoo** [comment]
Hey thanks :)
